salt/pillar/git_pillar.py

```diff
@@ -315,6 +315,12 @@ def ext_pillar(minion_id, repo, pillar_dirs):
             False
         )
         for pillar_dir, env in six.iteritems(pillar.pillar_dirs):
+            # If pillarenv is set, only grab pillars with that match pillarenv
+            if opts['pillarenv'] and env != opts['pillarenv']:
+                log.debug(
+                    'Env \'{0}\' does not match pillarenv \'{1}\', skipping pillar'.format(env,opts['pillarenv'])
```

Comment:
Pylint is going to complain about the lack of a space between arguments to str.format().